pytential/symbolic/stokes.py

```diff
@@ -469,30 +241,222 @@ def apply_stress(self, density_vec_sym, normal_vec_sym, dir_vec_sym,
         :arg normal_vec_sym: a symbolic vector variable for the normal vectors
             (outward facing normals at source locations).
         :arg dir_vec_sym: a symbolic vector for the application direction.
-        :arg mu_sym: a symbolic variable for the viscosity.
         :arg qbx_forced_limit: the *qbx_forced_limit* argument to be passed on
             to :class:`~pytential.symbolic.primitives.IntG`.
         """
+        raise NotImplementedError
+
+
+def _create_int_g(knl, deriv_dirs, density, **kwargs):
+    for deriv_dir in deriv_dirs:
+        knl = AxisTargetDerivative(deriv_dir, knl)
+
+    args = [arg.loopy_arg.name for arg in knl.get_args()]
+    for arg in args:
+        kwargs[arg] = var(arg)
+
+    res = sym.S(knl, density, **kwargs)
+    return res
+
+
+class StokesletWrapper(StokesletWrapperBase):
```

Comment:
```suggestion
class StokesletWrapper{Naive,Biharmonic}(StokesletWrapperBase):
```
Make `StokesletWrapper` a deprecated alias of ... `StokesletWrapperNaive`.